src/pages/workspace/categories/EditCategoryPage.tsx

```diff
@@ -46,7 +47,7 @@ function EditCategoryPage({route, policyCategories}: EditCategoryPageProps) {
                 >
                     <HeaderWithBackButton
                         title={translate('workspace.categories.editCategory')}
-                        onBackButtonPress={Navigation.goBack}
+                        onBackButtonPress={() => Navigation.goBack(ROUTES.WORKSPACE_CATEGORY_SETTINGS.getRoute(route.params.policyID, route.params.categoryName))}
```

Comment:
Please make sure that navigation transition animation is correct on native

Reply:
There's an issue where the last screen appears twice in the stack. Do we have a workaround for this?


https://github.com/Expensify/App/assets/24796318/4732a15b-5b9a-48f0-a86d-2655fa4bc790

Reply:
just for reference, this was meant to fix the issue with going back behavior on the web.
https://github.com/Expensify/App/pull/37969#discussion_r1525148144

Reply:
cc: [c3024](https://github.com/c3024)